src/components/HomeSearchBox/index.tsx

```diff
@@ -20,23 +40,58 @@ const HomeSearchBox = () => {
 
   const handleKeyPress = (event: React.KeyboardEvent<HTMLInputElement>) => {
     if (event.key === 'Enter') {
```

Comment:
When pressing `enter`, it always append the first search's suggestion. We don't need to do anything when the `searchValue` is empty right?
```suggestion
    if (searchValue && event.key === 'Enter') {
```